benchmarks/ekomark/navigator/navigator.py

```diff
@@ -0,0 +1,229 @@
+from ..banana_cfg import banana_cfg
+from ..data import db
+
+table_objects = bnav.table_objects
+table_objects["o"] = db.Operator
+
+class NavigatorApp(bnav.navigator.NavigatorApp):
+    """
+    Navigator base class holding all elementry operations.
+
+    Parameters
+    ----------
+        cfg : dict
+            banana configuration
+        mode : string
+            mode identifier
+    """
+
+    myname = "eko"
+    table_objects = table_objects
+
+    def fill_theories(self, theo, obj):
+        """
+        Collect important information of the theory record.
+
+        Parameters
+        ----------
+            theo : dict
+                database record
+            obj : dict
+                to be updated pandas record
+        """
+        for f in [
+            "PTO",
+            "XIR",
+            "ModEv",
+            "Q0",
+            "Qref",
+            "alphas",
+        ]:
+            obj[f] = theo[f]
+        obj["kcThr_mc"] = theo["mc"] * theo["kcThr"]
+        obj["kbThr_mb"] = theo["mb"] * theo["kbThr"]
+        obj["ktThr_mt"] = theo["mt"] * theo["ktThr"]
+
+    def fill_operators(self, op, obj):
+        """
+        Collect important information of the operator record.
+
+        Parameters
+        ----------
+            op : dict
+                database record
+            obj : dict
+                to be updated pandas record
+        """
+        xgrid = op["interpolation_xgrid"]
+        obj["xgrid"] = (
+            f"{len(xgrid)}pts: "
+            + f"{'log' if op['interpolation_is_log'] else 'x'}"
+            + f"^{op['interpolation_polynomial_degree']}"
+        )
+
+        obj["debug_skip_non_singlet"] = op["debug_skip_non_singlet"]
+        obj["ev_op_max_order"] = op["ev_op_max_order"]
+        obj["ev_op_iterations"] = op["ev_op_iterations"]
+        obj["Q2grid"] = op["Q2grid"]
+
+    def fill_cache(self, cac, obj):
+        """
+        Collect important information of the cache record.
+
+        Parameters
+        ----------
+            cac : dict
+                database record
+            obj : dict
+                to be updated pandas record
+        """
+        vals = cac["result"]["values"]
+        q2s = list(vals.keys())
+        # assume the vals are homogenous (true for bare eko results) and look
+        # only at the first one
+        pdfs = len(next(iter(vals.values())))
+
+        obj["operators"] = f"{pdfs} pdfs @ Q^2 {q2s} Gev^2"
+
+        obj["theory"] = cac["t_hash"][: self.hash_len]
+        obj["observables"] = cac["o_hash"][: self.hash_len]
+        for f in ["pdf", "external"]:
+            obj[f] = cac[f]
+
+    def fill_logs(self, lg, obj):
+        """
+        Collect important information of the log record.
+
+        Parameters
+        ----------
+            lg : dict
+                database record
+            obj : dict
+                to be updated pandas record
+        """
+        q2s = lg["log"].q2s
```

Comment:
@AleCandido do you have the implementation of .q2s still at head? I guess you just didn't commit - or should I reimplement a new? I guess it should be a unique sum of all children; note that I changed the key in the dfdict to give a proper name, so other things might be broken as well

Reply:
It should be in banana/develop:
https://github.com/N3PDF/banana/blob/f134fcb394ca65a4875ac1dfe873e2b7f96226d0/src/banana/data/dfdict.py#L72-L77

Maybe you didn't pull banana, is it possible?

Reply:
you are right